actions/complete/index.js

```diff
@@ -15,9 +15,9 @@ const { asFirefallAction } = require('../FirefallAction.js');
 
 async function main(params) {
   const {
-    prompt, temperature, model, firefallClient,
+    prompt, temperature, asJson, model, firefallClient,
   } = params;
-  return firefallClient.completion(prompt ?? 'Who are you?', temperature ?? 0.0, model ?? 'gpt-4');
+  return firefallClient.completion(prompt ?? 'Who are you?', temperature ?? 0.0, asJson ?? false, model ?? 'gpt-4-turbo');
```

Comment:
It would make sense to check if we always request JSON, and if so, I would default to JSON.

Reply:
I agree. The default should be set to JSON mode as this is the most common use case.